src/backend/launcher.ts

```diff
@@ -1019,6 +1020,13 @@ async function callRunner(
   // requires a "./"
   if (!isWindows) bin = './' + bin
 
+  // this must be a native sideload
```

Comment:
what does this comment mean? that at this point we are sure this is a native sideloaded game? I don't understand it

Reply:
`that at this point we are sure this is a native sideloaded game` yes that's what it means :)